processing/pyproject.toml

```diff
@@ -29,7 +36,12 @@ checkers = [
 ]
 
 [tool.ruff]
-target-version = "py311"
-select = ["E", "F", "W", "C", "N"]
-ignore = ["E501"]  # Example: Ignore line length errors
+target-version = "py313"
```

Comment:
It's funny because you've added pre-commit configuration but you didn't use it when commiting this file. If you did, you would see an error:
```
pre-commit error using ruff-check
Cause: TOML parse error at line 40, column 18
   |
40 | target-version = "py313"
   |                  ^^^^^^^
unknown variant py313,
```
That's caused by `ruff` not supporting py313 as of now. That said, we either remove the pre-commit file - as we only have ruff config there and ruff doesn't support our python version - or downgrade the project to py312 (which I would not advise).